Common/src/main/java/com/example/examplemod/CommonInit.java

```diff
@@ -8,7 +8,7 @@
 // import and access the vanilla codebase, libraries used by vanilla, and optionally third party libraries that provide
 // common compatible binaries. This means common code can not directly use loader specific concepts such as Forge events
 // however it will be compatible with all supported mod loaders.
-public class CommonInit {
+public class  CommonInit {
```

Comment:
Why are you adding double spacing here, and in other classes?

Reply:
I did not do this manually or intentionally. Probably some Idea formatting rule. I will ind what is causing it and remove.

Reply:
Fixed